public/orbs-public-beta-own-infra.md

```diff
@@ -0,0 +1,74 @@
+# Running Orbs v2 on your own infrastructure
+
+## Prerequisites
+
+* A Linux machine with Docker Swarm
+* Boyar v1.1.1 [binary download](https://s3.amazonaws.com/orbs-network-releases/infrastructure/boyar/boyar-v1.1.1.bin)
```

Comment:
why not latest? is this important?

Reply:
We don't have "latest" boyar, the versions are pinned.